exma-project/src/multipleExam.java

```diff
@@ -0,0 +1,71 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class MultipleExam { // ������ ��� ���
+	public static void main(String[] args) {
+		while (true) {
+			int input = 0;
+			System.out.println("1 : ������, 2 : ����, 3: ���");
+			System.out.print("���ڸ� �Է��ϼ��� : ");
+			Scanner sc = new Scanner(System.in); // ����ڰ� �Է��� ���ڸ� �޴� ���� ����� 1������ import�� �ʿ�
+			input = sc.nextInt();
+			if (input == 1) {
+				System.out.print("1~999������ ���ڸ� �Է����ּ��� : ");
+				int num1 = 0; // num1 ���� �ʱ�ȭ
+
+				// try-catch����� ����ó���ϱ� : ���ڰ� �ƴ� ���ڸ� �Է¹�� ��� ��α׷� ���
+				try {
+					num1 = sc.nextInt(); // �Է¹�� ���� int�� ���� �����Ѵ�.
+					if ((num1 > 999) || (num1 < 1)) // 1~999 ����� �Ѿ�� ���� �߻� �� ���� ó��
+						System.out.println("������ ����� ���� �ʽ�ϴ�. 1~999������ ���ڸ� �Է����ּ���");
+					else {
+						for (int i = 1; i <= 9; i++) { // ������ ����� ��ġ�� ��� ������ ���
+							System.out.println(num1 + " * " + i + " = " + num1 * i);
+						}
+					}
+				} catch (InputMismatchException e) {
+					// �Է¹�� ������ Ÿ�԰� ������ ������ Ÿ���� ���� ��� ��� �߻��ϴ� exception
+					System.out.println("���ڰ� �ƴմϴ�. ��α׷�� ����մϴ�.");
```

Comment:
숫자가 아닐 경우에 프로그램이 종료되지 않습니다
```suggestion
System.out.println("숫자가 아닙니다. 프로그램을 종료합니다.");
sc.close();
return;
```
을 추가해주시면 숫자가 들어오지 않았을 때
정상적으로 프로그램이 종료될 것 같아요!